source/blood/src/blood.cpp

```diff
@@ -455,8 +456,12 @@ void PreloadCache(void)
     gSysRes.PrecacheSounds();
     gSoundRes.PrecacheSounds();
     if (MusicRestartsOnLoadToggle)
+    {
```

Comment:
Please revert these braces and newlines

Reply:
sorry for all this. im still learning.

Reply:
sorry again.. i think i got it right this time. i quit trying the history and revert function. and i just copy the file from your repository and did the changes again.

Reply:
if you need me to delete this and start over the branch and pull request so all this commit trash is not kept in the history let me know